DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: iSEEfier
 Title: Streamlining the creation of initial states for starting an iSEE instance
-Version: 1.1.0
+Version: 1.3.0
```

Comment:
Ok, here we should still do the "correct bumping" - 1.1.1 is the version to edit this to.

We would use then in the NEWS the reference to 1.2.0 to simply say that it is in the release branch since that onwards.